agents/mutator.py

```
@@ -1,45 +1,62 @@
+
     if not stats:
         log(f"No stats found for {protocol_name}. Nothing to mutate.")
         return False
-    
-    failure_rate = stats['failure_rate']
-    log(f"Protocol {protocol_name}: {stats['failures']}/{stats['total_executions']} failures (rate: {failure_rate:.2%})")
-    
+
+    failure_rate = stats["failure_rate"]
+    log(
+        f"Protocol {protocol_name}: {
```

Pylint (3.3.7)

[error] Parsing failed: 'unterminated string literal (detected at line 27) (agents.mutator, line 27)'

(E0001)